packages/client-sdk-nodejs/test/integration/foo.test.ts

```diff
@@ -0,0 +1,36 @@
+import {CacheClient, CredentialProvider, Configurations} from '../../src';
```

Comment:
this file should be deleted.  I was using it to do some writes from the SDK and then test whether i could `get` the corresponding keys from the CLI.  This is how I confirmed that the collections were not being stored to the right keys.